index.js

```diff
@@ -3,16 +3,43 @@ new Vue({
   data: {
   	todos: [
   		{ text: 'test', done: false, id: Date.now() }
-  	]
+  	], 
+    activeTodos: [], 
+    completedTodos: [],
+    showActive: false,
+    showCompleted: false,
+    showAll: false
   },
   methods: {
   	addTodo({target}){
     	this.todos.push({text: target.value, done: false, id: Date.now()})
       target.value = ''
+      console.log(this.todos)
     },
     removeTodo(id) {
-    	console.log('removeTodo has been clicked')
     	this.todos = this.todos.filter(todo => todo.id !== id)
-    }
+    },
+    updateActiveAndCompletedLists() {
+      this.completedTodos = this.todos.filter(todo => todo.done == true)
+      this.activeTodos = this.todos.filter(todo => todo.done == false)
+      },

```

Comment:
} should be in line with function name `updateActiveAndCompletedLists`. fix the indentation here